src/client/test-run/index.js.mustache

```diff
@@ -0,0 +1,148 @@
+(function () {
+    if (window.self !== window.top)
+        return;
+
+    //API - should be defined here in local scope
+    var act                = null;
+    var ok                 = null;
+    var notOk              = null;
+    var eq                 = null;
+    var notEq              = null;
+    var handleAlert        = null;
+    var handleConfirm      = null;
+    var handlePrompt       = null;
+    var handleBeforeUnload = null;
+    var inIFrame           = null;
+    var __set$             = null;
+    var __get$             = null;
+    var __get$Loc          = null;
+    var __set$Loc          = null;
+    var __call$            = null;
+    var __proc$Script      = null;
+
+    var $      = null;
+    var jQuery = null;
+
+    //NOTE: Global __waitFor is used to ensure page is ready before the next step executing.
+    //Usage: call the global __waitFor function everywhere you want (test file, wrappers, modules etc..)
+    //The arguments are identical with the act.waitFor function arguments.
```

Comment:
The global __waitFor is used to ensure that the page is ready for the next step to be executed.
Usage: call the global __waitFor wherever required (test file, wrappers, modules, etc). The arguments are identical to the act.waitFor function arguments.